src/store.rs

```diff
@@ -0,0 +1,29 @@
+use crate::{async_trait, Result, Session};
+
+/// An async session backend.
+#[async_trait]
+pub trait SessionStore: std::fmt::Debug + Send + Sync + 'static + Clone {
```

Comment:
Convention, lifetime last, maybe Debug should be `use`'d
```suggestion
pub trait SessionStore:  Clone + std::fmt::Debug + Send + Sync + 'static {
```